### PR TITLE
[13.0][ADD] edi_voxel_stock_picking_secondary_unit: new module

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT28726

This module extends the functionality of 'Voxel stock picking' module to match secondary units of measure instead of standard units of measure during the stock picking sending process.

Note: I included the commit of the PR of the edi_voxel_secondary_unit (https://github.com/OCA/edi/pull/357) module until it is merged

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]
As I said in the other module, can you separate in two diferent commits the changes of edi_voxel_stock_picking from the addition of the new module??

**Comment by pedrobaeza** [comment]
In fact, both are including #357 as extra commit for having the CI green. You should review that one, got merged, and then this PR should be rebased.

**Comment by ernestotejeda** [comment]
> As I said in the other module, can you separate in two diferent commits the changes of edi_voxel_stock_picking from the addition of the new module??

@CarlosRoca13 changes done.

File: edi_voxel_stock_picking/reports/report_voxel_picking.py [hunk 1]
```diff
@@ -96,18 +96,22 @@ def _get_product_data(self, line):
             customer_sku = line.picking_id._get_customer_product_sku(
                 line.product_id, line.picking_id.sale_id.partner_invoice_id
             )
+        qty, mu = self._get_qty_mu(line)
```
Comment (pedrobaeza) [inline edi_voxel_stock_picking/reports/report_voxel_picking.py]: Don't add  new method, but override existing one `_get_product_data`

File: edi_voxel_stock_picking/tests/test_voxel_stock_picking.py [hunk 2]
```diff
@@ -44,78 +44,84 @@ def setUpClass(cls):
...
-                "note": "Picking note (test)",
-            }
-        )
-        cls.picking.move_lines[0].write({"quantity_done": 2})
-        cls.picking.move_lines[1].write({"quantity_done": 1})
-        cls.picking.move_lines[1].move_line_ids.lot_id = lot.id
-        cls.picking.button_validate()
+        return sale_order
 
     def test_get_voxel_filename(self):
```
Comment (pedrobaeza) [inline edi_voxel_stock_picking/tests/test_voxel_stock_picking.py:128]: Same problem of test executed twice.

File: edi_voxel_stock_picking_secondary_unit/README.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
+===========
+Module name
```
Comment (pedrobaeza) [inline edi_voxel_stock_picking_secondary_unit/README.rst]: This is not correct

File: edi_voxel_stock_picking_secondary_unit/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Voxel stock picking secondary unit",
+    "summary": "Map Voxel UoM to Stock Pickking Secondary UoM and Use Them",
```
Comment (pedrobaeza) [inline edi_voxel_stock_picking_secondary_unit/__manifest__.py]:
```suggestion
    "summary": "Export Secondary UoMs Voxel Code in picking Voxel documents",
```

File: edi_voxel_stock_picking_secondary_unit/reports/report_voxel_picking.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ReportVoxelPicking(models.AbstractModel):
+    _inherit = "report.edi_voxel_stock_picking.template_voxel_picking"
+
+    def _get_qty_mu(self, line):
```
Comment (pedrobaeza) [inline edi_voxel_stock_picking_secondary_unit/reports/report_voxel_picking.py]: The already commented about which method to override.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**